compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -532,6 +532,11 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
                     enum_candidates.into_iter().map(|(_variant_path, enum_ty_path)| enum_ty_path),
                     Applicability::MachineApplicable,
                 );
+                if self.r.session.features_untracked().associated_const_equality {
+                    err.help(format!(
```

Comment:
can this be a `span_suggestion`?

Reply:
changed it to be a `span_suggestion`, I've started feeling that the errors are too verbose but this may be fine since it'll be rarely hit.

Reply:
There are other suggestion modes, like tool only and short, if you think the suggestion is too verbose. This just ensures it's possible to be applied via rustfix.

Reply:
Hm well it's probably fine as is, I don't think I'll change it